h2o-py/h2o/backend/server.py

```diff
@@ -105,7 +105,7 @@ def start(jar_path=None, nthreads=-1, enable_assertions=True, max_mem_size=None,
         assert_is_type(log_dir, str, None)
         assert_is_type(log_level, str, None)
         assert_satisfies(log_level, log_level in [None, "TRACE", "DEBUG", "INFO", "WARN", "ERRR", "FATA"])
-        assert_is_type(max_log_file_size, str)
+        assert_is_type(max_log_file_size, str, None)
```

Comment:
What is this change?

Reply:
I have no idea--it's not in my github history...